api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -463,11 +523,13 @@ def stats(self) -> Stats:
     @property
     def shape(self) -> Tuple[int, int]:
         """
-        Get the shape of the data that will be returned by this ExperimentDataPipe. This is the number of
+        Get the shape of the data that will be returned by this ``ExperimentDataPipe``. This is the number of
         obs (cell) and var (feature) counts in the returned data. If used in multiprocessing mode
         (i.e. DataLoader instantiated with num_workers > 0), the obs (cell) count will reflect the size of the
         partition of the data assigned to the active process.
 
+        Args: None.
```

Comment:
I believe the guidelines allow you to leave this blank.

Reply:
removed!